spec/spec_helper.rb

```diff
@@ -134,7 +134,7 @@ def subject
   config.include(
     RSpec::LintExampleGroup,
     type: :lint,
-    file_path: Regexp.compile(['spec', 'unit', 'puppet-lint', 'plugins'].join('[\\\/]')),
+    file_path: Regexp.compile(['spec', 'puppet-lint', 'plugins'].join('[\\\/]')),
```

Comment:
Reverting this actually breaks the spec tests in puppet-lint.

Lets make it more inclusive of both patterns:

```suggestion
    file_path: Regexp.new('spec[\\\/](unit[\\\/])?puppet-lint[\\\/]plugins'),
```